core/server/api/canary/users.js

```diff
@@ -230,5 +233,31 @@ module.exports = {
                 return models.ApiKey.refreshSecret(model.toJSON(), Object.assign({}, {id: model.id}));
             });
         }
+    },
+
+    resetAllPasswords: {
+        headers: {
+            cacheInvalidate: true
```

Comment:
Why are we invalidating the cache?

Reply:
I couldn't find any docs on this property, I was thinking it would avoid the specific route from landing in the cache.
After looking through the Ghost Pro repositories, it looks like this header might clear the cache for the whole Ghost instance?

Reply:
> this header might clear the cache for the whole Ghost instance?

Yup, that's exactly what it does. We don't want to do that unless any of the content (posts, tags, etc.) has changed.